### ISSUE TITLE
Disable auto restart after plugin install by default

### ISSUE THREAD

**Issue by jjw24** [Flow-Launcher/Flow.Launcher#3028]
For new user experience, Flow shouldn't restart automatically after plugin installs, otherwise have to keep navigate back to the Plugin Store to continue installing more plugins.

Let's change the default behaviour and disable auto restart after plugin installation.

**Comment by Fludem** [comment]
I agree. I believe that setting an explicit behaviour should be removed entirely. 

Instead, the MessageBox should prompt the user to choose whether to restart now or later. 

There's no need to bury a setting within the settings of an individual plugin. Also, the choice to restart now or later is always situational.

For instance, enabling this setting could be helpful if it didn't require confirmation to restart and you were only doing a CRUD on one plugin. 
 
If I happen to be installing multiple plugins, disabling it would make the most sense. That would mean going out of my way to toggle it off before installing my new plugins and then back on once I was finished.
 
At this point, this feature is only causing me problems instead of solving any.

To wrap it back up, my points are:

- The setting is hidden away and not in a prominent location.

- Even if it were in a prominent location, would it be any easier than deciding on the spot if I want to restart now or later?
